lib/api/src/initial-state.ts

```diff
@@ -0,0 +1,55 @@
+import { location } from 'global';
+import merge from './lib/merge';
+
+import { themes } from '@storybook/theming';
+import { State } from './index';
+import { defaultShortcuts } from './modules/shortcuts';
+
+const initial: State = {
+  layout: {
+    isToolshown: true,
+    isFullscreen: false,
+    showPanel: true,
+    showNav: true,
+    panelPosition: 'bottom',
+  },
+
+  ui: {
+    name: 'STORYBOOK',
+    url: 'https://github.com/storybooks/storybook',
+    enableShortcuts: true,
+    sortStoriesByKind: false,
+    sidebarAnimations: true,
+    theme: themes.normal,
+  },
+
+  location,
+  path: '/',
+  viewMode: undefined,
+  storyId: undefined,
+
+  storiesHash: {},
```

Comment:
The modules should export these themselves

Reply:
If that's a problem for TS, then we need to solve that, rather than spreading the module over several files, which defeats the purpose of a module.